package.json

```diff
@@ -16,16 +16,13 @@
   "files": [
     "index.js"
   ],
-  "peerDependencies": {
-    "stylelint": "^13.7.1",
-    "stylelint-order": "^4.1.0",
-    "stylelint-scss": "^3.18.0",
-    "stylelint-selector-bem-pattern": "^2.1.0"
-  },
   "devDependencies": {
-    "stylelint": "^13.7.1",
-    "stylelint-order": "^4.1.0",
+    "stylelint": "^13.13.1",
```

Comment:
Not sure if we need this as these are dev dependencies of the order config package already. Regarding to the docs https://docs.npmjs.com/cli/v7/commands/npm-install devdependencies are installed by default if not in a production environment.